src/Symfony/Component/AssetMapper/ImportMap/ImportMapManager.php

```diff
@@ -345,14 +321,9 @@ private function addImplicitEntries(ImportMapEntry $entry, array $currentImportE
             return $currentImportEntries;
         }
 
-        // remote packages aren't in the asset mapper & so don't have dependencies
-        if ($entry->isRemotePackage()) {
-            return $currentImportEntries;
-        }
```

Comment:
An edge case (that does not currently apply) where vendor assets have relative imports, so we need to add those to the importmap.

More broadly, just another example of not treating the remote packages any differently.